rewriter/TEnum.cc

```diff
@@ -189,16 +190,29 @@ void TEnum::run(core::MutableContext ctx, ast::ClassDef *klass) {
     for (auto &stat : oldRHS) {
         if (auto enumsDo = asEnumsDo(stat)) {
             auto &block = ast::cast_tree_nonnull<ast::Block>(enumsDo->block);
+            vector<ast::TreePtr> newStats;
```

Comment:
why have intermediary? why not pass insSeqStats dirrectly?

Reply:
I got a type error, because ast::InsSeq::STATS_store isn't a valid `vector<ast::TreePtr>`, and I can't use `ast::InsSeq::STATS_store` in the helper's signature, because in the other half of the conditional, there's no InsSeq, there's just the `ast::ClassDef::RHS_store`.